packages/experience/src/hooks/use-prefilled-identifier.ts

```diff
@@ -0,0 +1,42 @@
+import { type SignInIdentifier } from '@logto/schemas';
+import { useContext, useMemo } from 'react';
+
+import UserInteractionContext from '@/Providers/UserInteractionContextProvider/UserInteractionContext';
+import { type IdentifierInputValue } from '@/components/InputFields/SmartInputField';
+
+import useLoginHint from './use-login-hint';
+
+type Options = {
+  enabledIdentifierType?: SignInIdentifier[];
```

Comment:
nit: Maybe `enabledIdentifiers` or `enabledSignInidentifiers`, as it's actually an array, should be better using plural form.

Reply:
Updated.